app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { Table } from "@bigbinary/neetoui";
+
+import { CONTACTS_COLUMNS as columns } from "./constants";
+import { CONTACTS_ROWS as rows } from "./constants";
```

Comment:
- Why are you renaming literal constants to lowercase names?
- Why are you not importing all entities in a single statement from same module? What's the point of splitting them out into two separate import statements?

Reply:
Sure, Done changes. Thanks